MobileCenterCrashes/MobileCenterCrashesTests/MSCrashesTests.mm

```diff
@@ -14,6 +14,7 @@
 #import "MSServiceAbstractPrivate.h"
 #import "MSServiceAbstractProtected.h"
 #import "MSUtil.h"
+#import "MSCrashesUtil.h"
```

Comment:
About your first comment:
Crashes is just looping over the buffered logs which should happen really fast, channel takes care of the prioritization. I don't think Crashes should care about what a priority actually means.

2nd comment: Adding one!